docs/developer-docs/dev-setup.md

```diff
@@ -36,11 +36,11 @@ VPN profiles expire after 12 hours. You'll need to download and connect with a n
 > **Please note** that you need to have your VPN running to access the Platform Hub and also talk to cluster APIs.
 
 You will need to register with the Platform Hub in order to gain access tokens for our Kubernetes clusters.
-Head to [Platform Hub], you will need your O365 credentials to login/sign up. You will be asked to connect your Github account to your Platform Hub account - this will give you access to our [BAU] board, which is used to raise requests and issues regarding the Platform or your project. More information can be found in the [developer documentation].
+Head to [Platform Hub], you will need your O365 credentials to login/sign up. You will be asked to connect your Github account to your Platform Hub account. This will give you access to project repositories under the UKHomeOffice Organisation in Github.
```

Comment:
```suggestion
Head to [Platform Hub], you will need your O365 credentials to login/sign up. You will be asked to connect your Github account to your Platform Hub account. This will give you access to project repositories under the UKHomeOffice Organisation in GitHub.
```